src/main/java/frc/robot/subsystems/DriveBaseSubsystem.java

```diff
@@ -70,14 +70,16 @@
   private NetworkTableEntry sbRightBiconsumerSpeed;
   private NetworkTableEntry sbLeftPosition;
   private NetworkTableEntry sbRightPosition;
+  private NetworkTableEntry sbGyroInfo;
 
   public DriveBaseSubsystem(Joystick joystick, boolean usingExternal) {
     driveBaseSpeed = 1;
     driverJoystick = joystick;
 
     motorControllers = new MotorController[4];
-    gyro = new AHRS(Port.kMXP);
-    gyro.reset(); // resets the heading of the robot to 0
+    gyro = new AHRS(SPI.Port.kMXP);
+    gyro.enableLogging(true);
```

Comment:
Not sure we want to enable logging during competition.  We could use the  Constants.DebugMode to only turn this on while we are competing.

( It is used for the shuffleboard tabs. )

Reply:
What are your thoughts Tony and Calvin?

Reply:
I wasn't sure what it did in the first place so I'll try testing the gyro without it.